package.json

```diff
@@ -333,7 +333,7 @@
     "@metamask/snaps-rpc-methods": "^12.3.0",
     "@metamask/snaps-sdk": "^6.24.0",
     "@metamask/snaps-utils": "^9.3.0",
-    "@metamask/solana-wallet-snap": "^1.30.3",
+    "@metamask/solana-wallet-snap": "^1.30.4",
```

Comment:
This bump is in 12.18.1 but the PR against main is merged late: https://github.com/MetaMask/metamask-extension/pull/33128